spec/active_model/entity/validations/exclusive_presence_validator_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe ActiveModel::Entity::Validations::ExclusivePresenceValidator do
+  subject(:dummy) { dummy_model.new(attributes) }
+
+  fields = %i[a b c]
+
+  let(:dummy_model) do
+    Class.new do
+      include ActiveModel::Model
+      include ActiveModel::Validations
+
+      attr_accessor :a, :b, :c
+
+      validates_with ActiveModel::Entity::Validations::ExclusivePresenceValidator, fields:
```

Comment:
```suggestion
      validates_with described_class, fields:
```

Reply:
nice try, but it won't work, because it's not in scope